__tests__/__snapshots__/from-schema-test.ts.snap

```diff
@@ -34,6 +32,9 @@ exports[`gql2ts Supports older TypeScript versions removes Nullability annotatio
     starship: IStarship;
     allVehicles: IVehiclesConnection;
     vehicle: IVehicle;
+    /*
+    description: Fetches an object given its ID
+  */
```

Comment:
indentation is off, the typescript prettifier doesn't seem to know what to do here... can look into https://github.com/prettier/prettier, but i think it's a bit heavy to include just for formatting comments. plus it's an automated tool, so if formatting isn't 100% i don't think it's an issue